doc/api/net.md

```diff
@@ -1125,7 +1125,7 @@ Returns true if input is a version 6 IP address, otherwise returns false.
 [Identifying paths for IPC connections]: #net_identifying_paths_for_ipc_connections
 [Readable Stream]: stream.html#stream_class_stream_readable
 [duplex stream]: stream.html#stream_class_stream_duplex
-[half-closed]: https://tools.ietf.org/html/rfc1122#section-4.2.2.13
+[half-closed]: https://tools.ietf.org/html/rfc1122
```

Comment:
Maybe replace with https://tools.ietf.org/html/rfc1122#page-87
@jasnell might have an idea how it is best to ref the RFC?

Reply:
I was thinking about the page hash, but it opens a doc with the wrong section in the top which also may be confusing.